drivers/mge-hid.c

```diff
@@ -832,28 +832,28 @@ static const char *eaton_input_bypass_check_range(double value)
     if ((bypass_low > 0 && bypass_high > 0) &&
         (bypass_voltage >= bypass_low && bypass_voltage <= bypass_high) &&
 		(bypass_frequency >= lower_frequency_limit && bypass_frequency <= upper_frequency_limit)) {
-        return "bypassOn"; /* Enter bypass mode */
+        return "On"; /* Enter bypass mode */
```

Comment:
Most other such statuses deal with lower-cased `on`/`off`, Can you please change this one to match? - least-surprise principle for consumers :)